templates/node/events/test/integration.js

```diff
@@ -21,7 +21,7 @@ test('Integration: handles a valid event', t => {
       .send({ message: 'hello' })
       .set(Spec.id, 'TEST-EVENT-1')
       .set(Spec.source, 'http://localhost:8080/integration-test')
-      .set(Spec.type, 'dev.faas.example')
+      .set(Spec.type, 'dev.function.example')
```

Comment:
Same as above. I should land https://github.com/boson-project/faas/pull/202, then if you can rebase on top of that and re-run pkger this should all be good.